week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr [2]; // complete this statement
```

Comment:
This works! It works because there are only 3 entries in the array. However it's worth thinking how you could make it work for any length of array (consider using the arr.length property).